sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -0,0 +1,1085 @@
+            **kwargs  # type: **Any
+    ):
+        # type: (...) -> CertificateOperation
+        """Creates a new certificate.
+
+        If this is the first version, the certificate resource is created. This
+        operation requires the certificates/create permission.
+
+        :param name: The name of the certificate.
+        :type name: str
+        :param policy: The management policy for the certificate.
+        :type policy:
+         ~azure.security.keyvault.certificates._models.CertificatePolicy
+        :param bool enabled: Determines whether the object is enabled.
+        :param tags: Application specific metadata in the form of key-value pairs.
+        :type tags: dict(str, str)
+        :returns: The created CertificateOperation
+        :rtype: ~azure.keyvault.certificates.models.CertificateOperation
+
+        Example:
+            .. literalinclude:: ../tests/test_examples_certificates.py
+                :start-after: [START create_certificate]
+                :end-before: [END create_certificate]
+                :language: python
+                :caption: Create a certificate
+                :dedent: 8
+        """
+
+        if enabled is not None:
+            attributes = self._client.models.CertificateAttributes(
+                enabled=enabled
+            )
+        else:
+            attributes = None
+
+        if not policy:
+            lifetime_actions = [LifetimeAction(
+                days_before_expiry=90,
+                action_type="AutoRenew"
+            )]
+            policy = CertificatePolicy(key_properties=KeyProperties(exportable=True,
+                                                                    key_type='RSA',
+                                                                    key_size=2048,
+                                                                    reuse_key=True,
+                                                                    key_usage=[
+                                                                        KeyUsageType.c_rl_sign,
+                                                                        KeyUsageType.data_encipherment,
+                                                                        KeyUsageType.digital_signature,
+                                                                        KeyUsageType.key_agreement,
+                                                                        KeyUsageType.key_cert_sign,
+                                                                        KeyUsageType.key_encipherment
+                                                                    ]),
+                                       issuer_name="Self",
+                                       lifetime_actions=lifetime_actions,
+                                       content_type=SecretContentType.PFX,
+                                       subject_name="CN=DefaultPolicy",
+                                       validity_in_months=12)
```

Comment:
In .NET I made the default policy configurable.  We could consider doing something like that possibly.  We don't need to update it in this PR, but perhaps open an issue to track this.